### PR TITLE
Gradle Enterprise is renamed to Develocity

### PR DESCRIPTION
**Pull request by runningcode**
This replaces Gradle Enterprise with Develocity in README only.

This does not rename code, code comments, build configuration or publishing descriptions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,31 +11,31 @@
...
 
 See [here](.mvn/extensions.xml) for an example.
 
 ### Version compatibility
 
-This table details the version compatibility of the Common Custom User Data Maven extension with the Gradle Enterprise Maven extension.
+This table details the version compatibility of the Common Custom User Data Maven extension with the Develocity Maven extension.
 
-| Common Custom User Data Maven extension versions | Gradle Enterprise Maven extension versions |
+| Common Custom User Data Maven extension versions | Develocity Maven extension versions |
```
Comment (erichaagdev) [inline README.md:38]: Can you reformat the table to account for the shorter column header?

**Review by erichaagdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (runningcode) [reply README.md]: good catch!

File: README.md [hunk 2]
```diff
@@ -11,31 +11,31 @@
 
 [![Verify Build](https://github.com/gradle/common-custom-user-data-maven-extension/actions/workflows/build-verification.yml/badge.svg?branch=main)](https://github.com/gradle/common-custom-user-data-maven-extension/actions/workflows/build-verification.yml)
 [![Maven Central](https://img.shields.io/maven-central/v/com.gradle/common-custom-user-data-maven-extension)](https://search.maven.org/artifact/com.gradle/common-custom-user-data-maven-extension)
-[![Revved up by Gradle Enterprise](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.solutions-team.gradle.com/scans)
+[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.solutions-team.gradle.com/scans)
```
Comment (erichaagdev) [inline README.md:14]:
```suggestion
[![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.solutions-team.gradle.com/scans)
```

**Review by erichaagdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (runningcode) [reply README.md]: Thanks

**Pull request merged**